imod/mf6/multimodel/exchange_creator_unstructured.py

```diff
@@ -70,8 +83,30 @@ def _compute_geometric_information(self) -> pd.DataFrame:
         Vj = centroid_2 - edge_coordinates[:, 0]
         length = np.linalg.norm(U, axis=1)
 
-        # The cross product of U and V equals the area of the parallellogram,
-        # dividing by length (the base of the parallellogram) gives the orthogonal distances cl1 & cl2.
+        # vertex index, coordinate index, point 1 or point 2
+        dx = edge_coordinates[:, 1, 0] - edge_coordinates[:, 0, 0]
+        dy = edge_coordinates[:, 1, 1] - edge_coordinates[:, 0, 1]
+
+        normal = np.array((dy[:], -dx[:]), dtype=np.float_).T
+        outward_vector = centroid_2 - centroid_1
+
+        # the inproduct between the edge normal and the outward vector is positive if they go in the same direction.
+        # we use this to swap the normal if needed so that it goes outward.
+        inprod = (
+            normal[:, 0] * outward_vector[:, 0] + normal[:, 1] * outward_vector[:, 1]
+        )
+
+        normal = np.where(
+            np.transpose((inprod[:] > 0, inprod[:] > 0)), normal[:], -normal[:]
+        )
+
+        # using the normal and the norm of the normal we can compute the angle of the normal
+        norm_normal = np.sqrt(normal[:, 0] * normal[:, 0] + normal[:, 1] * normal[:, 1])
```

Comment:
You can use linalg.norm, just like in line 84

Reply:
done